sanic/app.py

```diff
@@ -671,8 +671,8 @@ def run(self, host=None, port=None, debug=False, ssl=None,
         """
         # Default auto_reload to false
         auto_reload = False
-        # If debug is set, default it to true
-        if debug:
+        # If debug is set, default it to true (unless on windows)
+        if debug and os.name != 'posix':
```

Comment:
shouldn't this be `os.name == 'posix'`?

Reply:
:rofl: wow really thought I could handle a 2 line pr

Reply:
Happens to the best of us 😄 that's why review is good!